Techion.rte/Devices/Explosives/NanoSwarmGrenade/NanoSwarm.lua

```diff
@@ -108,40 +108,43 @@ function Update(self)
 
     if not MovableMan:IsActor(self.target) or self.target.Team == self.Team then
         --Find a target.
+		local shortestDist;
```

Comment:
Looks like some wonky whitespacing here. More accurately, the rest of the file seems to be using spaces. You should just convert it to tabs.

Reply:
Yeah this one was infested with those.